BraveWallet/Crypto/AssetIconView.swift

```diff
@@ -37,6 +38,14 @@ struct AssetIconView: View {
         return Image(uiImage: image)
       }
     }
+    
+    if network.isNativeAsset(token) {
+      // check bundled images
+      if UIImage(named: network.nativeTokenLogo, in: .current, with: nil) != nil {
+        return Image(network.nativeTokenLogo, bundle: .current)
```

Comment:
I think this should just be an `if let` and use the `Image(uiImage:)` initializer for `Image` so we don't have to load the image twice

Reply:
fixed in [7c8ecca](https://github.com/brave/brave-ios/pull/5772/commits/7c8ecca2ff5e599880a10a90dff9cfadcaffc4d9)